docs/dev-guide/python/index.mdx

```diff
@@ -8,101 +8,151 @@ toc_max_heading_level: 4
 keywords:
   - dev guide
   - python
+  - introduction
 tags:
   - dev-guide
   - python
 ---
 
 This guide is meant to provide a comprehensive overview of the structures, primitives, and features used in [Temporal Application](/temporal#temporal-application) development.
 
-## Project-setup
+## Core application
 
-The project setup section of the Temporal Python SDK Developer's guide covers the minimum set of concepts and implementation details needed to build and run a Temporal Application in Python—that is, all the relevant steps to start a Workflow Execution that executes an Activity.
+Develop basic Temporal application with Workflows and Activities in Python using Temporal SDK.
```

Comment:
These description sentences are really choppy... can we clean these up and copy edit them a bit?

> ... using Temporal SDK
> ... Explore using Signals in Temporal Python to